common/src/main/java/com/wynntils/features/chat/ChatMentionFeature.java

```diff
@@ -40,6 +41,9 @@ public class ChatMentionFeature extends Feature {
     @RegisterConfig
     public final Config<String> aliases = new Config<>("");
 
+    @RegisterConfig
+    public final Config<Boolean> suppressMentionsInInfo = new Config<>(true);
```

Comment:
```suggestion
    public final Config<Boolean> suppressMentionsInInfo = new Config<>(false);
```

Let's not change default behavior.

Reply:
Ah, sorry